src/hotspot/cpu/ppc/sharedRuntime_ppc.cpp

```diff
@@ -856,6 +858,8 @@ int SharedRuntime::c_calling_convention(const BasicType *sig_bt,
       // Although AIX runs on big endian CPU, float is in the most
       // significant word of an argument slot.
 #define FLOAT_WORD_OFFSET_IN_SLOT 0
+#elif defined(BSD)
```

Comment:
One more instance of BSD to convert to _ALLBSD_SOURCE

Reply:
done